packages/did/package.json

```diff
@@ -23,5 +23,8 @@
     "name": "EnergyWeb",
     "url": "https://www.energyweb.org/"
   },
-  "license": "GPL-3.0-or-later"
+  "license": "GPL-3.0-or-later",
+  "dependencies": {
```

Comment:
Should this be in the dev dependencies as it is being used only in tests?

Reply:
Right. Fixed